frontend/src/scenes/pipeline/hogfunctions/HogFunctionInputs.tsx

```diff
@@ -448,7 +460,12 @@ export function HogFunctionInputWithSchema({ schema }: HogFunctionInputWithSchem
                                         {...attributes}
                                         {...listeners}
                                     >
-                                        {schema.label || schema.key}
+                                        {schema.label || schema.key}{' '}
```

Comment:
Nit: that empty space could go in the conditional

Reply:
Wasn't even needed!